src/UnbxdSearchWrapper.js

```diff
@@ -125,6 +125,7 @@ class UnbxdSearchWrapper extends Component {
     componentDidMount() {
         const { unbxdCore } = this.state;
         const { trackCategory } = this.getAnalytics();
+        unbxdCore.isLoadedFromSpellCheck = false;
```

Comment:
can we instead expose a method to set values in the core instead of directly updating its state?